src/components/PInput.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <input v-on="$listeners"
+        :disabled="disabled"
+        :class="classNames"
+        :type="type"
+        :value="value"
+        >
+</template>
+
+<script>
+export default {
+  name: 'PInput',
+
+  props: {
+    type: {
+      type: String,
+      required: true
+    },
+
+    value: {
+      type: String,
+      default: ''
```

Comment:
We shouldn't default this, require please

Reply:
Done.